pkg/analyze/database_shared.go

```diff
@@ -0,0 +1,52 @@
+package analyzer
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/blang/semver"
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/troubleshoot/pkg/collect"
+)
+
+func compareDatabaseConditionalToActual(conditional string, result *collect.DatabaseConnection) (bool, error) {
+	parts := strings.Split(strings.TrimSpace(conditional), " ")
```

Comment:
I know this is not new, but `strings.Fields` is a better function to use for this.